### PR TITLE
Unify look and feel with project website

### PR DESCRIPTION
**Pull request by AMoo-Miki**
### Description
This change allows for the unification of documentation website and the project website. Documentation content would reside at `/docs` and be served via `https://opensearch.org/docs`.

 
### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -38,179 +38,172 @@
...
+                <li class="breadcrumb-nav-list-item"><span>{{ page.title }}</span></li>
+              </ol>
+            </nav>
+          {% endif %}
+        {% endunless %}
+        <div id="main-content" class="main-content" role="main">
+          {% if site.heading_anchors != false %}
+            {% include vendor/anchor_headings.html html=content beforeHeading="true" anchorBody="<svg viewBox=\"0 0 16 16\" aria-hidden=\"true\"><use xlink:href=\"#svg-link\"></use></svg>" anchorClass="anchor-heading" anchorAttrs="aria-labelledby=\"%html_id%\"" %}
+          {% else %}
+            <p class="warning" style="margin-top: 0">This documentation remains in a beta state. It has content gaps and might contain bugs.</p>
```
Comment (aetter) [inline _layouts/default.html:134]: This should be removed for launch. It's the top beta banner.

**Review by aetter — CHANGES_REQUESTED** [review]
The "on this page" section remains a problem here. If it's long, such as in this screenshot, it overflows off the screen and isn't usable. Even if it's short, it sticks in the middle of the screen rather than gracefully flowing up to the top on scroll down.

toc-bug

File: _layouts/default.html (hunk 1 above)
Comment (AMoo-Miki) [reply _layouts/default.html:134]: Done

File: _config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 title: OpenSearch documentation
 description: >- # this means to ignore newlines until "baseurl:"
   Documentation for OpenSearch, the Apache 2.0 search, analytics, and visualization suite with advanced security, alerting, SQL support, automated index management, deep performance analysis, and more.
-baseurl: "" # the subpath of your site, e.g. /blog
+baseurl: "/docs" # the subpath of your site, e.g. /blog
 url: "https://docs-beta.opensearch.org" # the base hostname & protocol for your site, e.g. http://example.com
```
[the remark this answers is not in the document]
Comment (AMoo-Miki) [reply _config.yml:5]: Done.

**Comment by AMoo-Miki** [comment]
Going in via `new-home`.

**Pull request closed without merging**